maintenance/migratestatus/migratestatus.go

```diff
@@ -123,7 +124,7 @@ func main() {
 		}
 	}
 
-	githubClient, err := o.github.GitHubClient(secretAgent, o.dryRun)
+	githubClient, err := o.github.GitHubClient(secretAgent, o.dryRun, componentField)
```

Comment:
We don't need to include fields that are already included via the DefaultFieldsFormatter.

Reply:
D'oh, I thought it had neither :|